src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -243,6 +243,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
 
     backendCall.instantiateCommand match {
       case Success(command) => runWithJes(backendCall, command, jesInputs, jesOutputs, runIdForResumption)
+      case Failure(ex: SocketTimeoutException) => throw ex // probably a GCS transient issue, throwing will cause it to be retried
```

Comment:
Yeah I agree this should be documented more.  The `FailedExecutionHandle` means the fate of the execution is definite and it is failed.  The failed `Future` is interpreted by the caller of this method as a condition which merits a retry.